src/Drupal/Migrate/MigrateMissingSourceRowsEvent.php

```diff
@@ -3,7 +3,7 @@
 namespace Drush\Drupal\Migrate;
 
 use Drupal\migrate\Plugin\MigrationInterface;
-use Symfony\Component\EventDispatcher\Event;
+use Symfony\Contracts\EventDispatcher\Event;
```

Comment:
Is 'contracts' OK for Symfony 4 and 5?

Reply:
Symfony 5.4 requires `"symfony/event-dispatcher-contracts": "^2|^3"`
Symfony 4.4 requires `"symfony/event-dispatcher-contracts": "^1.1"`

All `symfony/event-dispatcher-contracts` versions are exposing the Event class with the same namespace. See:

* https://github.com/symfony/event-dispatcher-contracts/blob/1.1/Event.php
* https://github.com/symfony/event-dispatcher-contracts/blob/2.3/Event.php
* https://github.com/symfony/event-dispatcher-contracts/blob/3.0/Event.php

Right now Drupal 9 doesn't allow to bump Symfony to 5, so we can't test. Anyway, maybe we should consider to run at least one test with `composer update --prefer-lowest` to check the lowest dependency versions scenario,